### ISSUE TITLE
what type of spatial attention(SA)?

### ISSUE THREAD

**Issue by leolin65** [ristea/sspcab#4]
Hi,

In your paper, could you explain what type of spatial attention(SA)? thank you very much.

**Comment by ristea** [comment]
The one used in the CBAM approach. PyTorch code snipped below.
`class SpatialAttention(nn.Module):
    def __init__(self, kernel_size=7):
        super(SpatialAttention, self).__init__()

        self.conv1 = nn.Conv2d(2, 1, kernel_size, padding=kernel_size//2, bias=False)
        self.sigmoid = nn.Sigmoid()

    def forward(self, x):
        avg_out = torch.mean(x, dim=1, keepdim=True)
        max_out, _ = torch.max(x, dim=1, keepdim=True)
        x = torch.cat([avg_out, max_out], dim=1)
        x = self.conv1(x)
        return self.sigmoid(x)`
